packages/runtime/datastore-definitions/src/channel.ts

```diff
@@ -98,7 +98,7 @@ export interface IDeltaConnection {
  */
 export interface IChannelStorageService {
     /**
-     * Reads the object contained at the given path. Returns a base64 string representation for the object.
+     * Reads the object contained at the given path. Returns a utf8 string representation for the object.
      */
     read(path: string): Promise<string>;
```

Comment:
I think this interface should closely mimic storage interfaces, i.e. have readBlob() & readString() methods (with read() eventually going away).